src/main/java/com/nerodesk/takes/TkApp.java

```diff
@@ -185,6 +185,10 @@ public void run() {
                 )
             ),
             new FkRegex(
+                "/admin",
+                new TkSecure(new TkGreedy(new TkAdmin(base)))
```

Comment:
@longtimeago TkGreedy is not needed here, it is only required when you are planning to read HTTP request body multiple times